src/js/ext-win-script.js

```diff
@@ -7,29 +7,39 @@
  */
 /* eslint-disable */
 (function($, Drupal) {
-  var $extLinkText = Drupal.t('external link opens in a new window / tab'),
-    $intLinkText = Drupal.t('opens in a new window / tab');
-  // Internal links with data-ext-url - turn them into external links
-  $("#main-content a[data-ext-type^='External']").each(function() {
-    $(this).attr('href', $(this).attr('data-ext-url'));
-    if ($(this).parent().hasClass('moreLink')) {
-      $(this).html('visit site');
+  Drupal.behaviors.nicsdruOriginsExtWin = {
+    attach: function (context) {
+      var $extLinkText = Drupal.t('external link opens in a new window / tab'),
```

Comment:
Just a small observation/question on `$` variable prefixes: looking at https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards#variablesandarrays and above section, you'd usually only prefix with a `$` when you're expecting to handle a jQuery object.